### PR TITLE
Deprecate parsing rules/ modeling rules using format issue fix

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7757

## Description
fix issue of deprecating parsing rules/ modeling rules using format. Where the non-supported tests: no tests field was added.

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
Link to failing ut [pipeline](https://app.circleci.com/pipelines/github/demisto/demisto-sdk/28403/workflows/139c2ac4-31f0-464e-8a9c-c77e69e938fa/jobs/252209) in the commit[168f6e6](https://github.com/demisto/demisto-sdk/pull/3634/commits/168f6e62e14454248bb42bfc7773d4bc3a3e26fa) after merger with master commit [9e93971](https://github.com/demisto/demisto-sdk/pull/3634/commits/9e93971be8a55fea84132f77df5e5851972465a8)

File: demisto_sdk/commands/format/tests/test_formatting_yml_test.py [hunk 1]
```diff
@@ -1531,6 +1531,66 @@ def test_update_deprecate_in_playbook(
...
+            ("", "Deprecated. No available replacement."),
+            ("Replacement entity", "Deprecated. Use Replacement entity instead."),
+        ],
+    )
+    def test_update_deprecate_in_modeling_rules(
+        self, pack, mocker, monkeypatch, user_input, description_result
+    ):
+        """
+        Given
+            - An playbook yml to deprecate.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/format/tests/test_formatting_yml_test.py:1546]:
```suggestion
            - A playbook yml to deprecate.
```
Let's address the two cases specifically (case a: no user input, Then: expted output should be blabla)

File: demisto_sdk/commands/format/tests/test_formatting_yml_test.py [hunk 2]
```diff
@@ -1531,6 +1531,66 @@ def test_update_deprecate_in_playbook(
...
+            ("", "Deprecated. No available replacement."),
+            ("Replacement entity", "Deprecated. Use Replacement entity instead."),
+        ],
+    )
+    def test_update_deprecate_in_parsing_rules(
+        self, pack, mocker, monkeypatch, user_input, description_result
+    ):
+        """
+        Given
+            - An playbook yml to deprecate.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/format/tests/test_formatting_yml_test.py:1576]: Same comment as above

File: demisto_sdk/commands/common/tests/structure_test.py [hunk 3]
```diff
@@ -939,3 +939,48 @@ def test_invalid_xsiam_report_missing_global_id(self, pack: Pack):
...
+        "deprecated": True,
+    }
+
+    @pytest.mark.parametrize(
+        "yml_data_update",
+        [DEPRECATED_RULE, DEPRECATED_WITH_COMMENT],
+    )
+    def test_deprecated_parsing_rule_is_valid(self, repo, yml_data_update):
+        """
+        Given:
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/structure_test.py:958]: Same comment as in test_formatting_yml_test.py for both tests here

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**